client/js/lounge.js

```diff
@@ -918,6 +924,23 @@ $(function() {
 		location.reload();
 	}
 
+	function updateDesktopNotificationStatus(){
+		var checkbox = $("#desktopNotifications");
+		var warning = $("#warnDisabledDesktopNotifications");
+
+		if (Notification.permission === "denied"){
+			checkbox.attr("disabled", true);
+			checkbox.attr("checked", false);
```

Comment:
I agree with @lpoujol here and I don't think `checkbox.indeterminate = true;` would help here. If the permission is denied, then is *must* be disabled and unchecked. Same for `default`.